tsconfig.json

```diff
@@ -25,6 +25,9 @@
   "include": [
     "./src",
     "node_modules/@types/jest/index.d.ts",
+    "node_modules/@types/jest/index.d.ts",
+    "node_modules/@types/jest/index.d.ts",
+    "node_modules/@types/jest/index.d.ts"
```

Comment:
this is actually already included above, did adding it three more times fix something for you?
I would not be very surprised if it did 🙃